### DISCUSSION TITLE
Time-based triggers?

### DISCUSSION THREAD

**Discussion by zidkim** [supabase/supabase#7646] [Questions] [answered]
Hi Team,

I'd like to be able to invoke functions involving supabase every Monday, Wednesday, and Friday.
Is there currently a reliable approach to doing this, or should I just wait for supabase Functions on August 1st?

Thanks in advance!

**Comment by steve-chavez** [comment, marked as answer]
You can use pg_cron for that. Check this blog post: https://supabase.com/blog/2021/03/05/postgres-as-a-cron-server

**Reply by zidkim** [reply]
Ah, thank you thank you.
I've yet to set this up but the article seems promising!

**Reply by zidkim** [reply]
Hey @steve-chavez,

Just had a chance to try this out.
I have a working API route in NextJS `api/trivia/startTrivia`.
I've also spun up an HTTP instance using NGROK so the URL is valid as well.

Following the steps outlined in the docs, I've setup the following below.
The HTTP request keeps returning a 404 not found...I saw on another post that there may be something wrong w the extension, but I don't think this pertains to me as requests have been made successfully, just getting a 404. 

```
select
  cron.schedule(
    'testo', -- name of the cron job
    '* * * * *', -- every minute
    $$
    select status
    from
      http_post(
        '....ngrok.io/api/trivia/startTrivia', -- webhook URL
        '{"hello": "world"}', -- payload
        'application/json'
      )
    $$
  );
```
